model/src/main/proto/example.proto

```diff
@@ -9,6 +9,10 @@ message UserAppHistory {
   bool already_opened_app = 1;
 }
 
+message OnBoardingFlow {
+  bool already_on_boarded_app = 1;
+  }
```

Comment:
Correct indentation.  Can we rename this file to something contextual.

Reply:
Corrected indentation. @BenHenning  please suggest the file name.